recipe/meta.yaml

```diff
@@ -1,5 +1,7 @@
-{% set version = "2.1.2" %}
-{% set build = 4 %}
+{% set pytorch_cpu_gpu = "pytorch-gpu" %}  # [cuda_compiler_version != "None"]
+{% set pytorch_cpu_gpu = "pytorch-cpu" %}  # [cuda_compiler_version == "None"]
```

Comment:
There's a bunch of migrator damage here, but this one should definitely be reduced (duplicated from lines 314-315)
```suggestion
```